test/index.html

```diff
@@ -75,8 +75,13 @@
  .catch(function(err) { console.error(err, err.stack); });
 
 var themes = document.querySelector('#themes');
+var currentLocation = window.location;
+var url = new URL(currentLocation);
 themes.addEventListener('change', function() {
-  theme = vegaThemes[themes.options[themes.selectedIndex].value];
+  theme = vegaThemes[themes.options[themes.selectedIndex].value]; 
+  url.searchParams.set('id', `${themes.selectedIndex}`);
+  url.searchParams.set('themes', `${themes.options[themes.selectedIndex].value}`);
```

Comment:
Why do we need id and themes?

Reply:
we need Id for setting the value in selection tab and themes for updating the themes on the page.

Reply:
What is a selection tab?

Reply:
Updating the value of a select tag. <select><select/> we can use only id but as you said that we have to include themes name in the url so I added the themes in url.

Reply:
When did I say we should include the theme name in the URL? I think a meaningful id is enough.